tool-populate/api/Questions.js

```diff
@@ -67,28 +43,29 @@ function createDemoQuestions(parentForm) {
     var newQuestion = new Question()
 
     var randomQuestionIndex = questionOrder[i]
+    Helpers.setAdminACL(newQuestion).then(function(){
+      newQuestion.set('text', DemoData.questions[randomQuestionIndex].text)
```

Comment:
variable scope of `var newQuestion` always points to the first iteration of the for loop. So only one question is created per form.